gateway/ln-gateway/src/envs.rs

```diff
@@ -37,10 +37,26 @@ pub const FM_LND_TLS_CERT_ENV: &str = "FM_LND_TLS_CERT";
 /// Necessary for LND configuration.
 pub const FM_LND_MACAROON_ENV: &str = "FM_LND_MACAROON";
 
-/// Environment variable that specifies the URL of an Esplora server. Necessary
-/// for LDK configuration.
+/// Environment variable that specifies the URL of an Esplora server.
+/// Necessary for LDK configuration if using esplora as the backend.
 pub const FM_LDK_ESPLORA_SERVER_URL: &str = "FM_LDK_ESPLORA_SERVER_URL";
 
+/// Environment variable that specifies the host of the bitcoind node.
+/// Necessary for LDK configuration if using bitcoind as the backend.
+pub const FM_LDK_BITCOIND_HOST: &str = "FM_LDK_BITCOIND_HOST";
```

Comment:
Do we want to use a one URL env var instead of 4 separate part env? Just like we do in Fedimint? (`FM_BITCOIN_RPC_URL`)

We could possibly make it fallback to the value of `FM_BITCOIN_RPC_URL` then as well, as these would usually be the same.

Reply:
I would be in favor of one URL env var. It would make the code below in `LightningBuilder::build` a bit cleaner

Reply:
Done.